test/api/v3/integration/tasks/PUT-tasks_id.test.js

```diff
@@ -176,6 +177,42 @@ describe('PUT /tasks/:id', () => {
       expect(savedDaily.reminders[0].id).to.equal(id1);
       expect(savedDaily.reminders[1].id).to.equal(id2);
     });
+
+    it('can set a shortName if the shortName does not already exist', async () => {
+      let savedDaily = await user.put(`/tasks/${daily._id}`, {
+        shortName: 'short-name',
+      });
+
+      expect(savedDaily.shortName).to.eql('short-name');
+    });
+
+    it('ignores shortName if it is already set', async () => {
```

Comment:
The behavior I decided on was to just ignore the shortName property if it is already set, rather than error. 

Ultimately, I still think it should be fine to update it.